### DISCUSSION TITLE
Natural frequency extraction

### DISCUSSION THREAD

**Discussion by senpaibernd** [OpenRadioss/OpenRadioss#676] [Q&A] [answered]
Is it already possible to extract natural frequencies (eigensolver on bord?) and an analysis based on it (e.g. frequency response analysis)?

**Comment by nawijn** [comment, marked as answer]
I don't know whether or not the pre-compiled binaries support ARPACK, but in general the answer would be "yes". You can configure eigenvalue extraction by means of the "/EIG" keyword. See the corresponding [documentation](https://2022.help.altair.com/2022.1/hwsolvers/rad/topics/solvers/rad/eig_starter_r.htm#eig_starter_r__eig_starter_r_fn_amg_rtz_rz)

**Reply by senpaibernd** [reply]
Great! I will try it tomorrow. Thank you for the answer.

**Reply by MarianBulla** [reply]
It is not possible to provide the direct usage of /EIG functionality in OpenRadioss, yet. 
We have to work on another solution/workaround, here.
As mentioned by @nawijn , ARPACK is a possible way to go.

**Reply by senpaibernd** [reply]
It would be great if there will be a guide on how to compile ARPACK with the engine 😃

**Comment by nawijn** [comment]
If it only requires compiling and linking to the appropriate versions of
OpenRadioss I am more than willing to give it a try. If it requires changes
in the underlying Fortran code, than it is more problematic for me.

Marco

**Reply by FGarelli** [reply]
Are there any updates on the natural frequency extraction with OpenRadioss?
